site/en/guide/tpu.ipynb

```diff
@@ -421,6 +421,7 @@
         "# scope, so that the variables can be mirrored on each device.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.      model.distribute_reduction_method = "sum"
</pre><p>This one shouldn't be here for TPU as per https://github.com/keras-team/keras/pull/16664#issuecomment-1195453876</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>